### PR TITLE
fix(swap): Fix price input format

### PR DESCRIPTION
**Pull request by jorbuedo**
related: https://emurgo.atlassian.net/browse/YOMO-825

### CODE REVIEW

File: apps/wallet-mobile/src/features/Swap/useCases/StartSwapScreen/CreateOrder/EditLimitPrice.tsx [hunk 1]
```diff
@@ -24,6 +23,7 @@ export const EditLimitPrice = () => {
   const {createOrder, limitPriceChanged} = useSwap()
   const sellTokenInfo = useTokenInfo({wallet, tokenId: createOrder.amounts.sell.tokenId})
   const buyTokenInfo = useTokenInfo({wallet, tokenId: createOrder.amounts.buy.tokenId})
+  const denomination = (sellTokenInfo.decimals ?? 0) - (buyTokenInfo.decimals ?? 0)
```
Comment (michalszorad) [inline apps/wallet-mobile/src/features/Swap/useCases/StartSwapScreen/CreateOrder/EditLimitPrice.tsx:26]: Will this work if sell token and buy token have different decimals?

Comment (jorbuedo) [reply apps/wallet-mobile/src/features/Swap/useCases/StartSwapScreen/CreateOrder/EditLimitPrice.tsx:26]: If they're different? Yes
If buy decimals are higher than sell decimals? Not sure... checking... maybe needs an absolute.

Comment (stackchain) [reply apps/wallet-mobile/src/features/Swap/useCases/StartSwapScreen/CreateOrder/EditLimitPrice.tsx:26]: I'd say that the denomination of the price should be fixed.

Comment (jorbuedo) [reply apps/wallet-mobile/src/features/Swap/useCases/StartSwapScreen/CreateOrder/EditLimitPrice.tsx:26]: It is right, denomination can be negative here

Comment (jorbuedo) [reply apps/wallet-mobile/src/features/Swap/useCases/StartSwapScreen/CreateOrder/EditLimitPrice.tsx:26]: > I'd say that the denomination of the price should be fixed.

Can't be fixed, depends on the pair you're trading

Comment (michalszorad) [reply apps/wallet-mobile/src/features/Swap/useCases/StartSwapScreen/CreateOrder/EditLimitPrice.tsx:26]: I think I understand now. The `createOrder.marketPrice` and `createOrder.limitPrice` are computed without domination, and the domination is added later for display

**Review by michalszorad — APPROVED** [review]

**Pull request merged**